editor/editor_help.cpp

```diff
@@ -99,6 +100,10 @@ const Vector<String> classes_with_csharp_differences = {
 };
 #endif
 
+static const String nbsp = String::chr(160);
+static const String nbsp_equal_nbsp = nbsp + "=" + nbsp;
+static const String colon_nbsp = ":" + nbsp;
```

Comment:
```suggestion
static constexpr String nbsp = String::chr(160);
static constexpr String nbsp_equal_nbsp = nbsp + "=" + nbsp;
static constexpr String colon_nbsp = ":" + nbsp;
```
Preferred for global constants.

Reply:
`String` has no `constexpr` constructors.